src/index.ts

```diff
@@ -133,55 +134,49 @@ async function hmacSha256Verify(key: Buffer, msg: Buffer, sig: Buffer): Promise<
  * Generate a new valid private key. Will use the window.crypto or window.msCrypto as source
  * depending on your browser.
  */
-export const generatePrivate = function (): Buffer {
+export const generatePrivate = function (): Uint8Array {
   let privateKey = randomBytes(32);
   while (!isValidPrivateKey(privateKey)) {
     privateKey = randomBytes(32);
   }
   return privateKey;
 };
 
-export const getPublic = function (privateKey: Buffer): Buffer {
+export const getPublic = function (privateKey: Uint8Array): Uint8Array {
   // This function has sync API so we throw an error immediately.
   assert(privateKey.length === 32, "Bad private key");
   assert(isValidPrivateKey(privateKey), "Bad private key");
   // XXX(Kagami): `elliptic.utils.encode` returns array for every
   // encoding except `hex`.
-  return Buffer.from(ec.keyFromPrivate(privateKey).getPublic("array"));
+  return secp256k1.getPublicKey(privateKey, false);
```

Comment:
remove comment.

also: does this return expected format? i remember elliptic was sometimes using 64 bytes pub key? while noble probably outputs 65 byte sec1.